src/ucp/wireup/wireup.c

```diff
@@ -676,6 +731,13 @@ static ucs_status_t ucp_wireup_msg_handler(void *arg, void *data,
         ucp_wireup_process_request(worker, ep, msg, &remote_address);
     } else if (msg->type == UCP_WIREUP_MSG_REPLY) {
         ucp_wireup_process_reply(worker, ep, msg, &remote_address);
+    } else if (msg->type == UCP_WIREUP_MSG_EP_CHECK) {
+        ucs_assert(msg->dst_ep_id != UCP_EP_ID_INVALID);
```

Comment:
can we remove the assert and use `UCP_WIREUP_MSG_CHECK(msg, ep, UCP_WIREUP_MSG_EP_CHECK);` inside the handler?